.github/workflows/dapr-release-notes.yml

```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-release-notes
+
+on:
+  repository_dispatch:
+    types: release_notes
+jobs:
+  build:
+    name: Generate release notes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Generate release notes
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+        run: python ./.github/scripts/generate_release_notes.py
+      - name: Commit and push to repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+          COMMIT_MSG: |
+            Generating Dapr release notes.
+            skip-checks: true
+        run: |
+          git config user.email "Dapr Bot"
```

Comment:
Direct commit to master seems risky without review.

Can we create PR and then let approvers or maintainers review and merge the pr ? like [dependent bot](https://github.com/dependabot/dependabot-core)

Reply:
I agree.

Reply:
Done. See sample here: https://github.com/artursouza/dapr/pull/5